compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -120,6 +120,7 @@ fn to_llvm_tls_model(tls_model: TlsModel) -> llvm::ThreadLocalMode {
         TlsModel::LocalDynamic => llvm::ThreadLocalMode::LocalDynamic,
         TlsModel::InitialExec => llvm::ThreadLocalMode::InitialExec,
         TlsModel::LocalExec => llvm::ThreadLocalMode::LocalExec,
+        TlsModel::EmulatedTls => llvm::ThreadLocalMode::GeneralDynamic,
```

Comment:
I tried mapping to `NotThreadLocal`, but this causes LLVM to not generate thread local symbol.